stack/indexer/indexer-security-init.sh

```diff
@@ -64,17 +64,24 @@ getNetworkHost() {
     HOST="${HOST//$NH}"
     HOST=$(echo "${HOST}" | tr -d "[\"\']")
 
+    isIP=$(echo "${HOST}" | grep -P "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$")
+    isDNS=$(echo "${HOST}" | grep -P "^[a-zA-Z0-9][a-zA-Z0-9-]{1,61}[a-zA-Z0-9](?:\.[a-zA-Z]{2,})+$")
+
     # Allow to find ip with an interface
-    PATTERN="^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$"
-    if [[ ! "${HOST}" =~ ${PATTERN} ]]; then
+    if [ -z "${isIP}" ] && [ -z "${isDNS}" ]; then
         interface="${HOST//_}"
-        HOST=$(ip -o -4 addr list "${interface}" | awk '{print $4}' | cut -d/ -f1)
+        HOST=$(ip -o -4 addr list "${interface}" 2>/dev/null | awk '{print $4}' | cut -d/ -f1) 
```

Comment:
I'm not a fan of hiding errors. Could you remove `2>/dev/null`?

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1823/commits/96c401a2e6da538a1cc4ba04f5039d5b4d1b6c54